net/tor-hs/files/nextcloud-update.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+# This script disables trusted domain check for given
+# hidden service address
+
+NEXTCLOUD_CLI_SCRIPT="/srv/www/nextcloud/occ"
+
+nextcloud_cli() {
+	sudo -u nobody php-cli "$NEXTCLOUD_CLI_SCRIPT" "$@"
+}
+
+
+nextcloud_add_domain() {
+	onion="$1"
+	if [ ! -z "$onion" ]; then
```

Comment:
I assume this can be replaced with [ -n ] && { }

Reply:
Fixed